resolwe_bio/tools/expression_aggregator.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python2
```

Comment:
All new scripts should be in Python 3, unless there is a good reason not to.

Reply:
The shebang line will be changed when the docker image with statistical python packages is ready.